include/ola/Clock.h

```diff
@@ -111,6 +112,12 @@ class BaseTimeVal {
                 struct timeval *result) const;
 
   void Set(int64_t interval_useconds);
+
+  /**
+   * @brief Sets m_tv
```

Comment:
A different comment would probably be better documentation. Perhaps something like this.
```suggestion
   * @brief Sets the value with a timespec.
```

Reply:
I agree :)